crates/driver/src/main.rs

```diff
@@ -181,6 +184,14 @@ fn read_all_project_conf(input_path: &Path, working_directory: &Path) -> Result<
     Ok(v)
 }
 
+pub fn to_directory(rel_path: &String) -> String {
```

Comment:
nit: I think we can make this `(rel_path: String) -> String` and avoid the clone. It looks like our use case can handle taking ownership. Also, since we are unconditionally calling clone, if the caller only has a `&str` or `&String` they can call `.clone()` and recover the efficiency of this implementation.

Reply:
Addressed in abefc9531c9940200281fa63f42e3a35011d47f0